docs/docs/limits/README.md

```diff
@@ -7,56 +7,39 @@ next: false
 
 Pipedream imposes limits on source and workflow execution, the events you send to Pipedream, and other properties. You'll receive an error if you encounter these limits. See our [troubleshooting guide](/troubleshooting/) for more information on these specific errors.
 
-Some of these limits apply only on the free tier. For example, Pipedream limits the daily number of invocations and execution time you can use on the free tier. **On paid tiers, you can run an unlimited number of invocations, for any amount of execution time**.
+Some of these limits apply only on the free tier. For example, Pipedream limits the daily number of credits and active workflows you can use on the free tier. **On paid tiers, you can run an unlimited number of credits per day, for any amount of execution time**.
 
-Other limits apply to both the free and paid tiers, but many can be raised upon request. Please see the details on each limit below.
+Other limits apply across the free and paid tiers. Please see the details on each limit below.
 
 **These limits are subject to change at any time**.
 
 [[toc]]
 
 ## Number of Workflows
 
-**You can run an unlimited number of workflows**, as long as each operates under the limits below.
+Free accounts are limited to 3 active workflows. [Paid tiers](https://pipedream.com/pricing) are able to activate more workflows.
```

Comment:
Limit is 10 to start

Reply:
Is there a new pricing matrix?

I referenced the **v0 Implementation** matrix in the Pipedream pricing doc:

https://docs.google.com/spreadsheets/d/1Hkx4M1Eo5L2WNWO9F2X6VVRJp3y6JThH4J4UMjQlT54/edit#gid=477812140